### PR TITLE
Fixing #1900 JS SDK file upload

### PR DESCRIPTION
**Pull request by blacha**
JS SDK file upload uses req.body._ContentType to specify the upload content type

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
@blacha updated the pull request.

**Comment by facebook-github-bot-4** [comment]
@blacha updated the pull request.

**Comment by facebook-github-bot-8** [comment]
@blacha updated the pull request.

**Comment by flovilmart** [comment]
Awesome!

File: src/middlewares.js [hunk 1]
```diff
@@ -84,6 +84,10 @@ function handleParseHeaders(req, res, next) {
         info.masterKey = req.body._MasterKey;
         delete req.body._MasterKey;
       }
+      if (req.body._ContentType) {
+        req.headers['content-type'] = req.body._ContentType;
+        delete req.body_contentType;
```
Comment (drew-gross) [inline src/middlewares.js]: Shouldn't this be `body._ContentType`?

**Comment by flovilmart** [comment]
Yep you're right! I'll patch

**Pull request merged**